### DISCUSSION TITLE
parameters on server restart

### DISCUSSION THREAD

**Discussion by STyx2909** [Vdauphin/HeartsAndMinds#1507] [Question and answer]
Am I wrong, assuming on server restart, default parameters are loaded?
If so, some kind of parameters save would be needed, instead of having to tweak parameters file.

**Comment by Vdauphin** [comment]
Hello,

> default parameters are loaded?

Yes, see https://community.bistudio.com/wiki/Mission_Parameters

> If so, some kind of parameters save would be needed, instead of having to tweak parameters file.

I don't see the point because:
- Most mission parameter have limited effect after the mission is saved
    - changing the density of enemes after saving will only affect city where player never entered
- Also overwriting Mission Parameter is not possible. Then player will be confused to see in loby a parameter not applied. 

Cheers

**Reply by STyx2909** [reply]
Thank you. I was focused on the time speed. I like at 3 or less. Would it be resetted at 5 on startup?
